src/xbot/common/subsystems/BaseConnectionMonitorSubsystem.java

```diff
@@ -9,17 +10,18 @@
 import java.util.Observable;
 import java.util.Observer;
 
-public abstract class BaseConnectionMonitorSubsystem extends BaseSubsystem implements Observer {
+@Singleton
+public class BaseConnectionMonitorSubsystem extends BaseSubsystem implements Observer {
```

Comment:
Since this isn't abstract, you probably should remove "Base."

If you imagine this being used just for the driver station, I'd call it "DriverStationConnectionMonitorSubsystem" or something like that.